ros_gz_sim/launch/gz_sim.launch.py.in

```diff
@@ -126,7 +127,9 @@ def launch_gz(context, *args, **kwargs):
     else:
         on_exit = None
 
-    return [ExecuteProcess(
+    return [
+        LogInfo(f"Launching gazebo with the environment variables: {env}"),
```

Comment:
I'm not sure if it's always safe to print the environment variables. Maybe be it can be enabled via a launch config?

Reply:
How about using a `LogDebug`? So by default it wont be enabled but if we set the log level to debug the information is shown.

Reply:
I'd prefer to be more explicit. Consider the scenario that ROS log messages are being sent to a remote logging service. If `env` contains credentials, then this is a potential leak.